src/CheckoutSdk/Payments/Links/PaymentLinkResponse.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+using System;
+
+namespace Checkout.Payments.Links
+{
+	public sealed class PaymentLinkResponse : Resource, IEquatable<PaymentLinkResponse>
+	{
+
+		public string Id { get; set; }
+
+		[JsonProperty(PropertyName = "expires_on")]
+		public string ExpiresOn { get; set; }
```

Comment:
Please check if it can be `DateTime`

Reply:
This field is String on file PaymentLinkResponse.java